crates/orchestrator/src/jobs/mod.rs

```diff
@@ -135,11 +135,36 @@ pub async fn create_job(
         .await
         .map_err(|e| JobError::Other(OtherError(e)))?;
     if existing_job.is_some() {
+        config
+            .alerts()
+            .send_alert_message(format!(
+                "Error in creating job. Job already exists. Internal ID : {:?}, Metadata : {:?}",
+                internal_id, metadata
+            ))
+            .await
+            .map_err(|e| JobError::Other(OtherError(e)))?;
         return Err(JobError::JobAlreadyExists { internal_id, job_type });
     }
 
     let job_handler = factory::get_job_handler(&job_type).await;
-    let job_item = job_handler.create_job(config.as_ref(), internal_id, metadata).await?;
+    let job_item = job_handler.create_job(config.as_ref(), internal_id.clone(), metadata.clone()).await;
+
+    // Error handling for sending alerts
+    let job_item = match job_item {
+        Ok(job_item) => job_item,
+        Err(e) => {
+            config
```

Comment:
instead of adding alerts at individual errors (because this way we'll have a lot), how about we just throw an error at the queue level for now. because very function in our code is triggered by a queue consumption